services/collectibles/api.go

```diff
@@ -165,3 +165,15 @@ func (api *API) ContractOwner(ctx context.Context, chainID uint64, contractAddre
 	}
 	return owner.String(), nil
 }
+
+func (api *API) AddTokenOwners(ctx context.Context, chainID uint64, contractAddress string, walletAddresses []string, amount int) error {
+	if len(walletAddresses) == 0 {
+		return errors.New("wallet addresses list is empty")
+	}
+	if amount <= 0 {
+		return errors.New("amount is <= 0")
```

Comment:
Since you use those error message twice, can you add them as constants at the top.

Reply:
Done